packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react'
+import { StyleSheet, ScrollView, Text, View, TouchableOpacity } from 'react-native'
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import fontStyles from '@celo/react-components/styles/fonts'
+import colors from '@celo/react-components/styles/colors'
+import VerificationComplete from 'src/icons/VerificationComplete'
+import VerificationDenied from 'src/icons/VerificationDenied'
+import VerificationPending from 'src/icons/VerificationPending'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { KycStatus } from 'src/account/reducer'
+import { useTranslation } from 'react-i18next'
+import PersonaButton from 'src/account/Persona'
+interface Props {
+  kycStatus: KycStatus | undefined
+}
+
+function LinkBankAccountScreen({ kycStatus }: Props) {
+  return (
+    <SafeAreaView style={styles.body}>
+      <ScrollView
+        contentContainerStyle={styles.contentContainer}
+        showsVerticalScrollIndicator={false}
+      >
+        <StepOne kycStatus={kycStatus} />
+        <StepTwo disabled={!kycStatus || kycStatus !== KycStatus.Completed} />
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function StepOne({ kycStatus }: Props) {
+  const { t } = useTranslation()
+
+  switch (kycStatus) {
+    case KycStatus.Created:
+      /* TODO: Ask Nitya on 12/1 about UI for Created Flow. I'll do it in separate PR */
```

Comment:
They should be very similar to the others, verifying with Nitya tomorrow morning.